exporter_test.go

```diff
@@ -24,7 +24,12 @@ import (
 	dto "github.com/prometheus/client_model/go"
 
 	"github.com/prometheus/statsd_exporter/pkg/clock"
+	"github.com/prometheus/statsd_exporter/pkg/event"
+	"github.com/prometheus/statsd_exporter/pkg/exporter"
```

Comment:
it seems strange to me that the exporter _tests_ are not inside the exporter package. Could you move those that make sense in too, and move the others to the appropriate place? it's fine to have program-wide tests in the main package but I would like them to be named appropriately then.